elasticsearch/helpers/actions.py

```diff
@@ -221,6 +223,7 @@ def _process_bulk_chunk(
     bulk_data,
     raise_on_exception=True,
     raise_on_error=True,
+    ignore=(),
```

Comment:
Instead of `ignore` let's use `ignore_status`, this is what the parameter will be called on APIs in 8.x onwards and will be deprecated in 7.x.